src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.cs

```diff
@@ -277,8 +277,13 @@ public static void LogWarning (object log, string msg, params object [] args)
 #if MSBUILD
 		public static bool IsMonoAndroidAssembly (ITaskItem assembly)
 		{
+			// NOTE: look for both MonoAndroid and Android
 			var tfi = assembly.GetMetadata ("TargetFrameworkIdentifier");
-			if (string.Compare (tfi, "MonoAndroid", StringComparison.OrdinalIgnoreCase) == 0)
+			if (tfi.IndexOf ("Android", StringComparison.OrdinalIgnoreCase) != -1)
```

Comment:
At this point in time (2024), this looks like it'll work.  However, this doesn't quite feel "future-proof", as it will succeed for *any* string that contains `Android`: `AreAndroidsHuman`, [`HarmonyOSIsMaybeAndroidCompatible`](https://www.osnews.com/story/139424/the-only-viable-android-and-ios-competitor-intends-to-leave-china-and-go-global/), etc.

Requiring that this be `== 0` will require that things *start with* `Android`, which might still be problematic, but "feels" nominally safer.

Reply:
We still support building agains `MonoAndoid` libraries, I think it will say `MonoAndroid` for old Xamarin.Android NuGet packages.